test/GroupByTest.cpp

```diff
@@ -1676,23 +1664,24 @@ TEST_F(GroupByOptimizations, computeGroupByObjectWithCount) {
   // Group by subject.
   auto getId = makeGetId(qec->getIndex());
   {
-    IdTable result{qec->getAllocator()};
     auto groupBy = GroupBy{qec, variablesOnlyX, aliasesCountX, xyScan};
-    ASSERT_TRUE(groupBy.computeGroupByObjectWithCount(&result));
-    ASSERT_EQ(result, makeIdTableFromVector(
-                          {{getId("<x>"), I(4)}, {getId("<z>"), I(1)}}));
+    auto result = groupBy.computeGroupByObjectWithCount();
+    ASSERT_TRUE(result.has_value());
+    ASSERT_EQ(result.value(), makeIdTableFromVector({{getId("<x>"), I(4)},
+                                                     {getId("<z>"), I(1)}}));
   }
 
   // Group by object.
   {
-    IdTable result{qec->getAllocator()};
     auto groupBy = GroupBy{qec, variablesOnlyY, aliasesCountY, yxScan};
-    ASSERT_TRUE(groupBy.computeGroupByObjectWithCount(&result));
-    ASSERT_EQ(result, makeIdTableFromVector({{getId("\"A\""), I(1)},
-                                             {getId("\"alpha\""), I(1)},
-                                             {getId("\"älpha\""), I(1)},
-                                             {getId("\"Beta\""), I(1)},
-                                             {getId("\"zz\"@en"), I(1)}}));
+    auto result = groupBy.computeGroupByObjectWithCount();
+    ASSERT_TRUE(result.has_value());
+    ASSERT_EQ(result.value(),
+              makeIdTableFromVector({{getId("\"A\""), I(1)},
+                                     {getId("\"alpha\""), I(1)},
+                                     {getId("\"älpha\""), I(1)},
+                                     {getId("\"Beta\""), I(1)},
+                                     {getId("\"zz\"@en"), I(1)}}));
```

Comment:
Same here, not even the explicit `result` variable is needed.

Reply:
It is - again because gtest tries to copy stuff